fastf1/core.py

```diff
@@ -2376,6 +2376,22 @@ def get_weather_data(self) -> pd.DataFrame:
             return pd.concat(wd, axis=1).T
         else:
             return pd.DataFrame(columns=self.session.weather_data.columns)
+        
+    def pick_lap(self, lap_number: int) -> "Laps":
+        """Return all laps of a specific LapNumber in self based on the driver's
+        three letters identifier or based on the driver number ::
```

Comment:
That's not correctly documented here. The driver number or identifier has nothing to do with picking the lap.

Reply:
Fixed it now